deployment/pcm/values.yaml

```diff
@@ -0,0 +1,105 @@
+### -------------- Naming -------------------
+# used in 
+# - common label: app.kubernetes.io/name otherwise "Chart name"
+# - also in selectorLabels together with release.name
+# defaults to "Chart.name"
+nameOverride: ""        
+# Used as daemonset name (usually based on truncated "name + release name")
+fullnameOverride: ""
+
+### -------------- Image options ------------
+image:
+  repository: ghcr.io/opcm/pcm
```

Comment:
please change the repo name to ghcr.io/intel/pcm
it has been made public and the legacy ghcr.io/opcm/pcm will be deleted soon

Reply:
done